code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -219,14 +219,6 @@
 
 	return 0
 
-
-/mob/living/carbon/human/has_legs()
-	. = 0
-	if(has_limb("r_foot") && has_limb("r_leg"))
-		.++
-	if(has_limb("l_foot") && has_limb("l_leg"))
-		.++
```

Comment:
what a proc

Reply:
Yeah it's handled somewhere else instead, in the poorly-named update_body, need to find a better solution

Reply:
It was just simpler than creating a new handler and putting it in all 27 places side by side. Ideally we get proper updates when limbs are made able/unable directly but i didn't think adding more to scope untangling this was a good idea at the time